pkg/cloudprovider/providers/vsphere/vsphere_test.go

```diff
@@ -39,7 +39,7 @@ func configFromEnv() (cfg VSphereConfig, ok bool) {
 	cfg.Global.Password = os.Getenv("VSPHERE_PASSWORD")
 	cfg.Global.Datacenter = os.Getenv("VSPHERE_DATACENTER")
 	cfg.Network.PublicNetwork = os.Getenv("VSPHERE_PUBLIC_NETWORK")
-	cfg.Global.Datastore = os.Getenv("VSPHERE_DATASTORE")
+	cfg.Global.DefaultDatastore = os.Getenv("VSPHERE_DATASTORE")
```

Comment:
This will be a deprecated. I did not quite understand why you have modified this test file.

Reply:
when I ran make test it failed.
I just fixed the make test for now. Even if it is deprecated, make test need to pass whenever pull request for kubernetes/master would be created.